src/assets/content.tsx

```diff
@@ -147,7 +147,8 @@ const content: Content = {
   whatsNew: {
     title: 'Gieß den Kiez Wartung:',
     description: [
-      'Nach dem Herbstputz ist uns aufgefallen, dass auch unsere Datenbank etwas aufgeräumt werden muss. Daher werden wir am 29.11.2022 von 11:00 bis 16:00 Uhr die Plattform für Wartungsarbeiten abschalten. Wir bitten um dein Verständnis.',
+      'Nach dem Herbstputz ist vor dem Frühjahrsputz. Daher werden wir am Montag den 9. Januar 2023 ab 10:00 Uhr an unseren DNS Einträgen und dem Nameserver herumdoktern. Die Plattform wird dann kurzzeitig nicht erreichbar sein.',
+      'Leider lässt sich bei solchen Arbeiten nicht genau sagen, wie lange die Plattform offline ist. Wir werden euch auf unserem Slack-Channel informieren, sobald die Plattform wieder erreichbar ist. Wir bitten um euer Verständnis.',
```

Comment:
Vielleicht statt "solchen Arbeiten": DNS Veränderungen brauchen immer unterschiedlich lang und es ist uns unmöglich die Dauer mit Genauigkeit einzuschätzen. ...

So mache wir deutlich dass es nicht mit unser Arbeit zu tun hat sondern mit der DNS? Feel free to ignore!